PayCalculatorAPI/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace PayCalculatorAPI
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+
```

Comment:
I think this is code that was generated for you, but nonetheless it's useful to know that `DateTime` is no longer the recommended way to represent a date and time in C#. Instead you should go for the `DateTimeOffset` class that largely replaces it, except in a few odd scenarios. 

See [DateTime vs DateTimeOffset](https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime) for more, but in a nutshell the new one is designed to handle time zones better.

Reply:
Alright thank you.